AUTHORS

```diff
@@ -868,5 +867,128 @@ Super Zheng <superzheng@tencent.com>
 Rafał Pocztarski <r.pocztarski@gmail.com>
 Michael Ruddy <mruddybtc@gmail.com>
 Andy Bettisworth <andy.bettisworth@accreu.com>
+Jörg Krause <joerg.krause@embedded.rocks>
+Alexander Martin <alex@suitupalex.com>
+Prince J Wesley <princejohnwesley@gmail.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+ronkorving <ron@ronkorving.nl>
+Xotic750 <xotic750@gmail>
+Nicholas Young <nicholas@nicholaswyoung.com>
+chrisjohn404 <chris.s.johnson4@gmail.com>
```

Comment:
Using my name is fine, Chris Johnson, the email is correct.

On Mon, Apr 25, 2016 at 10:16 AM, Сковорода Никита Андреевич <
notifications@github.com> wrote:

> In AUTHORS <https://github.com/nodejs/node/pull/6373#discussion_r60941491>
> :
>
> > @@ -868,5 +867,128 @@ Super Zheng <superzheng@tencent.com>
> >  Rafał Pocztarski <r.pocztarski@gmail.com>
> >  Michael Ruddy <mruddybtc@gmail.com>
> >  Andy Bettisworth <andy.bettisworth@accreu.com>
> > +Jörg Krause <joerg.krause@embedded.rocks>
> > +Alexander Martin <alex@suitupalex.com>
> > +Prince J Wesley <princejohnwesley@gmail.com>
> > +janriemer <janriemer@users.noreply.github.com>
> > +Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
> > +Martin von Gagern <Martin.vGagern@gmx.net>
> > +Hideki Yamamura <hideki.yamamura@gmail.com>
> > +Lenny Markus <lmarkus@paypal.com>
> > +Nelson Pecora <nelson@yoshokatana.com>
> > +ronkorving <ron@ronkorving.nl>
> > +Xotic750 <xotic750@gmail>
> > +Nicholas Young <nicholas@nicholaswyoung.com>
> > +chrisjohn404 <chris.s.johnson4@gmail.com>
>
> /cc @chrisjohn404 <https://github.com/chrisjohn404>, is that the
> preferred name?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6373/files/cf0ff99bf4a4f7cb2a84ad2544a2043bdd56f59d#r60941491>
>